src/assemble_workflow/bundle.py

```diff
@@ -132,7 +135,11 @@ def __get_min_dist(self, build_components: BuildComponents) -> Dist:
         min_dist_path = self._copy_component(min_bundle, "dist")
         logging.info(f"Copied min bundle to {min_dist_path}.")
         min_path = f"{self.build.filename}-{self.build.version}".replace("-SNAPSHOT", "")
-        min_dist = Dist.from_path(min_bundle.name, min_dist_path, min_path)
+        print("min_bundle.name: " + min_bundle.name)
```

Comment:
If you need to log something use `logging`, remove print

Reply:
That is for debugging, removed now.